### PR TITLE
RELEASE: Setup for v0.7.0

### LINKED ISSUE #695: Organise a new release `v0.7` with new features
Organise a new release for the latest round of PR's.

### PR DESCRIPTION
**Pull request by mmcky**
This PR sets up for `v0.7.0` release

fixes #695 

@oyamad tried to get the balance right re: listing the fixes and maintenance in a concise way while highlight the main fixes.

### CODE REVIEW

**Review by oyamad — CHANGES_REQUESTED** [review]
@mmcky Could you please also include #690 (which implements a Numba-jitted linear complementarity problem (LCP) solver based on Lemke's algorithm).

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+
+### Fixes and Maintenance
+
+1. [FIX: Fix period of reducible MarkovChain with custom state_values](https://github.com/QuantEcon/QuantEcon.py/pull/684) ([oyamad](https://github.com/oyamad))
+
+There were a range of additional maitenance and fixes including [Update __Iss.py - Remove broken link](https://github.com/QuantEcon/QuantEcon.py/pull/692), [693](https://github.com/QuantEcon/QuantEcon.py/pull/693), [689](https://github.com/QuantEcon/QuantEcon.py/pull/689), [Bring estimate_mc into top level namespace](https://github.com/QuantEcon/QuantEcon.py/pull/688), [Warn only when n in not int in tauchen](https://github.com/QuantEcon/QuantEcon.py/pull/673), [FIX: Avoid bare 'except'](https://github.com/QuantEcon/QuantEcon.py/pull/686), [FIX: DOC: Remove 2-byte spaces](https://github.com/QuantEcon/QuantEcon.py/pull/685), [MAINT: Import from _graph_tools, _gridtools](https://github.com/QuantEcon/QuantEcon.py/pull/683)
```
Comment (oyamad) [inline CHANGELOG.md:14]: This `[MAINT: Import from _graph_tools, _gridtools](https://github.com/QuantEcon/QuantEcon.py/pull/683)` is just a leftover from #682; it doesn't have to be included here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## Ver 0.7.0 (24-February-2023)
+
+### New
+
+1. [END: Add function fit_discrete_mc](https://github.com/QuantEcon/QuantEcon.py/pull/681) ([jstac](https://github.com/jstac)), ([Smit-create](https://github.com/Smit-create)) and ([oyamad](https://github.com/oyamad)) which takes a time series and fits a finite markov chain
+2. [ENH: Adding Discrete Approximation of VAR Methods](https://github.com/QuantEcon/QuantEcon.py/pull/640) ([crondonm](https://github.com/crondonm)) which is a routine based on the work `Finite-State Approximation Of VAR Processes: A Simulation Approach'' by Stephanie Schmitt-Grohé and Martín Uribe, July 11, 2010.
```
Comment (oyamad) [inline CHANGELOG.md:8]: I don't know how much we based our implementation on their work.

**Comment by mmcky** [comment]
thanks @oyamad made those updates and changes

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
+
+### Fixes and Maintenance
+
+1. [FIX: Fix period of reducible MarkovChain with custom state_values](https://github.com/QuantEcon/QuantEcon.py/pull/684) ([oyamad](https://github.com/oyamad))
+
+There were a range of additional maitenance and fixes including [Update __Iss.py - Remove broken link](https://github.com/QuantEcon/QuantEcon.py/pull/692), [693](https://github.com/QuantEcon/QuantEcon.py/pull/693), [689](https://github.com/QuantEcon/QuantEcon.py/pull/689), [Bring estimate_mc into top level namespace](https://github.com/QuantEcon/QuantEcon.py/pull/688), [Warn only when n in not int in tauchen](https://github.com/QuantEcon/QuantEcon.py/pull/673), [FIX: Avoid bare 'except'](https://github.com/QuantEcon/QuantEcon.py/pull/686), [FIX: DOC: Remove 2-byte spaces](https://github.com/QuantEcon/QuantEcon.py/pull/685)
+
+Thank you to ([bensonarafat](https://github.com/bensonarafat)), ([crondonm])(https://github.com/crondonm)), ([oyamad](https://github.com/oyamad)), ([jstac](https://github.com/jstac)), and ([Smit-create](https://github.com/Smit-create)) for all your contributions, PR reviews, and comments.
```
Comment (oyamad) [inline CHANGELOG.md:17]:
```suggestion
Thank you to ([bensonarafat](https://github.com/bensonarafat)), ([crondonm](https://github.com/crondonm)), ([oyamad](https://github.com/oyamad)), ([jstac](https://github.com/jstac)), and ([Smit-create](https://github.com/Smit-create)) for all your contributions, PR reviews, and comments. 
```

**Review by oyamad — APPROVED** [review]
@mmcky I think it is ready to tag a new release. Thanks!

**Comment by oyamad** [comment]
Updated CHANGELOG.md and rebased on the latest `main`.

**Pull request merged**